src/ConfigProvider.php

```diff
@@ -15,14 +15,29 @@ final class ConfigProvider
     public function __invoke(): array
     {
         return [
-            'dependencies'       => $this->getDependencies(),
-            'view_manager'       => [
-                'encoding' => 'utf-8',
-            ],
+            'dependencies' => $this->getDependencies(),
+            /**
```

Comment:
Since we are introducing them now, can we document the type of additional keys in this array at type level?